pkg/cmd/util/clientcmd/factory.go

```diff
@@ -510,12 +510,12 @@ func NewFactory(clientConfig kclientcmd.ClientConfig) *Factory {
 			if t.Spec.Paused {
 				return true, nil
 			}
-			t.Spec.Paused = true
+			body := []byte("{\"spec\":{\"paused\":true}}")
```

Comment:
calculate patches means if the JSON serialization ever changes, we don't have to sweep our codebase for little embedded JSON patches